client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/hooks/use-pattern-pages.ts

```diff
@@ -82,18 +83,34 @@ const usePatternPages = (
 		pageSlugs = INITIAL_PAGES;
 	}
 
-	pages = pageSlugs.map( ( slug ) => pagesMapByCategory[ slug ]?.[ 0 ] ).filter( Boolean );
+	pages = pageSlugs
+		.map( ( slug ) => {
+			const firstPage = pagesMapByCategory[ slug ]?.[ 0 ];
+			if ( firstPage ) {
+				return {
+					...firstPage,
+					title: getPagePatternTitle( firstPage ),
```

Comment:
This is required to fix the page title in the preview.

<img width="377" alt="Screenshot 2567-01-08 at 16 25 12" src="https://github.com/Automattic/wp-calypso/assets/1881481/f97edad6-a6f0-4baf-a83d-00591b4d452e">